packages/utils/user/phishingBanner.ts

```diff
@@ -8,8 +8,8 @@ const hidePhishingBannerAtom = atom(
   (get) => {
     const now = dayjs()
     const last = dayjs(get(phishingBannerAtom)).add(1, 'day')
-    const notPreview = process.env.NEXT_PUBLIC_VERCEL_ENV !== 'preview'
-    return last ? now.unix() > last.unix() && notPreview : notPreview
+    // const notPreview = process.env.NEXT_PUBLIC_VERCEL_ENV !== 'preview'
```

Comment:
Do we want to revert this or is this an intended change?

Reply:
don't have a strong opinion on this. Both looks fine to me

Reply:
Okay

Reply:
The preview environment doesn't display the info stripe, which may not be a good choice. Let's remove it for now and see.